### PR TITLE
upgrade/test-utils

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

File: test/TestDNSSEC.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const types = require('dns-packet/types');
 
 var dnssec = artifacts.require("./DNSSECImpl");
 const Result = require('@ensdomains/dnsprovejs/dist/dns/result')
+const { exception } = require('@ensdomains/test-utils')
```
Comment (Arachnid) [inline test/TestDNSSEC.js]: `exception` is a bit  too generic, can we use something else?

File: test/TestDNSSEC.js [hunk 2]
```diff
@@ -66,18 +67,12 @@ async function verifyFailedSubmission(instance, data, sig, proof) {
...
 
     try{
-        var tx = await instance.submitRRSet(data, sig, proof);
-    }
-    catch(error){
-        // @TODO use: https://github.com/ensdomains/root/blob/master/test/helpers/Utils.js#L8
-        // Assert ganache revert exception
-        assert.equal(error.message, 'Returned error: VM Exception while processing transaction: revert');
+        await instance.submitRRSet(data, sig, proof);
+    } catch(error){
```
Comment (Arachnid) [inline test/TestDNSSEC.js]: This won't work as desired - the test will still pass if no exception is thrown.

Comment (decanus) [reply test/TestDNSSEC.js]: No, due to line 75

File: test/TestDNSSEC.js (hunk 1 above)
Comment (decanus) [reply test/TestDNSSEC.js]: do you have a better name?

**Pull request closed without merging**